configure.ac

```diff
@@ -201,6 +201,76 @@ AC_ARG_ENABLE([certs],
     [AS_HELP_STRING([--enable-certs],[Enable X.509 cert support (default: disabled)])],
     [ENABLED_CERTS=$enableval],[ENABLED_CERTS=no])
 
+# wolfsently support
```

Comment:
Spelling - "wolfsently" should be "wolfsentry"

Reply:
thanks. fixed